### ISSUE TITLE
Incompatibility with first person model

### ISSUE THREAD

**Issue by Ryzhikkk** [cobrasrock/Skin-Swapper#60]
Infinity loop when starting minecraft, and I know that this got fixed in 3.1.4 but will it be fixed in 3.1.2?

**Comment by Chamelllemon** [comment]
No, that's why there are newer versions, I don't think anyone fixes bugs in older versions
